blueeyes/src/test/scala/quasar/RCValueGenerators.scala

```diff
@@ -25,20 +25,12 @@ trait RCValueGenerators {
 
   def genColumn(size: Int, values: Gen[Array[CValue]]): Gen[List[Seq[CValue]]] = containerOfN[List,Seq[CValue]](size, values.map(_.toSeq))
 
-  private def genNonArrayCValueType: Gen[CValueType[_]] =
+  def genCValueType: Gen[CValueType[_]] =
     Gen.oneOf[CValueType[_]](CString, CBoolean, CLong, CDouble, CNum,
       COffsetDateTime, COffsetTime, COffsetDate,
       CLocalDateTime, CLocalTime, CLocalDate, CInterval)
 
-  def genCValueType(maxDepth: Int = maxArrayDepth): Gen[CValueType[_]] = {
```

Comment:
can you explain why this can be deleted?

Reply:
It's not used; I copied this change from your PR.

Reply:
hmm, I didn't delete anything in this file. I think what's deleted here are useful additions (more complete generators).

Reply:
That's odd. I don't know why that would've happened. But `CArray` is not something useful to generate.